easybuild/easyconfigs/g/GROMACS/GROMACS-2018-CrayGNU-17.08-PLUMED-2.4.2-cuda-8.0.eb

```diff
@@ -0,0 +1,45 @@
+# contributed by Luca Marsella (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2018"
+cudaversion = '8.0'
+versionsuffix = '-PLUMED-2.4.2-cuda-%s' % cudaversion
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'usempi': 'True', 'openmp': 'True'}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+preconfigopts = ' pushd ../gromacs-%(version)s && plumed-patch -p --runtime --shared <<< 2 && popd && '
+configopts = ' -DCMAKE_C_FLAGS="-g -fPIC" -DCMAKE_CXX_FLAGS="-g -fPIC" -DFFTWF_INCLUDE_DIR=$FFTW_INC -DFFTWF_LIBRARY=$FFTW_DIR/libfftw3f.so '
+skipsteps = ['test']
+
+# CMake and libxml2 dependencies with dummy toolchain
+builddependencies = [
+    ('CMake', '3.9.1', '', True),
+    ('cudatoolkit/%s.61_2.4.3-6.0.4.0_3.1__gb475d12' %cudaversion, EXTERNAL_MODULE),
+    ('fftw/3.3.4.10', EXTERNAL_MODULE),
+    ('libxml2', '2.9.4', '', True),
+]
+
+dependencies = [
+    ('PLUMED', '2.4.2'),
```

Comment:
If `plumedversion` is defined, then we can avoid to hard code the version of the dependency:
```
    ('PLUMED', '%s' % plumedversion) 
```